fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -69,6 +77,25 @@ const RollCallOpened = () => {
     }
   };
 
+  const handleEnterManually = (input: string) => {
```

Comment:
It will be nice if this function has its test too

Reply:
It's now done